pydm/widgets/base.py

```diff
@@ -767,6 +771,28 @@ def ctrl_limit_changed(self, which, new_limit):
         else:
             self._lower_ctrl_limit = new_limit
 
+    def alarm_limit_changed(self, which: str, new_limit: float) -> None:
```

Comment:
Would you consider a string Enum for this?

e.g.,

```python
class AlarmLimit(str, enum.Enum):
    hihi = "HIHI"
    high = "HIGH"
    ...
```
It constrains `str` to one of 4 values which is rather nice

Reply:
Agreed, this is cleaner. Addressed here: e586e4a460a83f7853bf48c2e1a9b231c5311290